### PR TITLE
feat: convert etl lambdas to ECS

### PR DESCRIPTION
**Pull request by CalvinRodo**

### CODE REVIEW

File: env/staging/ecs/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+inputs = {
+  subnet_id                 = dependency.network.outputs.private_subnet_id
+  sg_id                     = dependency.network.outputs.csv_etl_sg_id
+  unmasked_metrics_s3_arn   = dependency.s3.outputs.unmasked_metrics_arn
+  masked_metrics_s3_arn     = dependency.s3.outputs.masked_metrics_arn
+  masked_metrics_bucket     = dependency.s3.outputs.masked_metrics_id
+  unmasked_metrics_bucket   = dependency.s3.outputs.unmasked_metrics_id
+  csv_etl_repository_url    = dependency.ecr.outputs.create_csv_repository_url
+  create_csv_repository_arn = dependency.ecr.outputs.create_csv_repository_arn
+  billing_tag_key           = "CostCentre"
```
Comment (patheard) [inline env/staging/ecs/terragrunt.hcl:52]: Aren't we able to just take `billing_tag_key` and `billing_tag_value` directly from `env/stating/env_vars.hcl`?

**Review by patheard — CHANGES_REQUESTED** [review]
See comments

File: env/staging/ecs/terragrunt.hcl (hunk 1 above)
Comment (CalvinRodo) [reply env/staging/ecs/terragrunt.hcl:52]: Nope was the issue I had this morning, unless all the modules have the variable it fails.

File: aws/ecs/ecs.tf [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  billing_tag_key              = var.billing_tag_key
+  billing_tag_value            = var.billing_tag_value
+  cluster_id                   = aws_ecs_cluster.in_app_metrics.id
+  subnet_id                    = var.subnet_id
+  sg_id                        = var.sg_id
+  template_file                = file("task-definitions/metrics.json")
+  vars = {
+    image                 = "hello-world"
+    awslogs-region        = "ca-central-1"
+    awslogs-stream-prefix = "ecs-unmasked-metrics"
```
[the remark this answers is not in the document]
Comment (CalvinRodo) [reply aws/ecs/ecs.tf:30]: yes good catch

**Review by patheard — APPROVED** [review]
👍🏻

**Pull request merged**